scrapy/extensions/throttle.py

```diff
@@ -2,6 +2,9 @@
 
 from scrapy.exceptions import NotConfigured
 from scrapy import signals
+from enum import Enum
+from datetime import datetime, timedelta
+from typing import List
```

Comment:
:lipstick: These should be under `import logging`, and sorted alphabetically:

```suggestion
from datetime import datetime, timedelta
from enum import Enum
from typing import List
```